rpm/SPECS/docker-ce.spec

```diff
@@ -79,7 +81,7 @@ export DOCKER_GITCOMMIT=%{_gitcommit}
 mkdir -p /go/src/github.com/docker
 ln -s /root/rpmbuild/BUILD/src/engine /go/src/github.com/docker/docker
 
-pushd engine
+pushd /root/rpmbuild/BUILD/src/engine
```

Comment:
nit: can't we use an RPM variable here, like `$RPM_BUILD_ROOT` ?

Reply:
I'll look at some cleaning up in a follow-up 👍